Dockerfile.build

```diff
@@ -0,0 +1,21 @@
+#~~~~ To build:
+# docker build -t kiwix/kiwix-lib-build -f Dockerfile.build .
+# docker create --name kiwix-lib-build kiwix/kiwix-lib-build
+# docker cp kiwix-lib-build:/usr/kiwix/build/src/libkiwix.a .
+
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get -y install sudo build-essential apache2 curl git wget pkg-config
+
+COPY . /usr/kiwix/
+WORKDIR /usr/kiwix/
+
+RUN apt-get install -y
```

Comment:
Does this apt-get install -y do anything productive? From a quick test it seems not be be harmful, however it also doesn't install any packages.